docs/sharing.md

```diff
@@ -401,10 +401,92 @@ Receive a sharing request.
 
 Answer a sharing request.
 
+
 ### DELETE /sharings/:id
```

Comment:
Could you use `sharing-id` instead of `id`? I think this could be confusing with the `_id`.